src/Engine/Audio/SteamAudio.cpp

```diff
@@ -1,15 +1,16 @@
 #include "SteamAudio.hpp"
 
 SteamAudio::SteamAudio() {
-
+    context = nullptr;
+    environmentalRenderer = nullptr;
 }
 
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Member variable 'SteamAudio::context' is not initialized in the constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AuninitMemberVar)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Member variable 'SteamAudio::environmentalRenderer' is not initialized in the constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AuninitMemberVar)